### PR TITLE
Add we've moved to README

### PR DESCRIPTION
**Pull request by MaxWang-MS**
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# &#x1F6A9;The docs have moved!
```
Comment (keveleigh) [inline README.md:1]: Are we able to embed the flag emoji directly? Or is there a benefit to using the `&#x1F6A9;` code?

```suggestion
# 🚩 The docs have moved!
```

Comment (MaxWang-MS) [reply README.md:1]: Oh I did not know we can embed it directly!

**Review by keveleigh — APPROVED** [review]

**Pull request merged**